integration-tests/hibernate-configuration/pom.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-integration-tests-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-hibernate-configuration-test-jpa</artifactId>
+    <name>Quarkus - Integration Tests - JPA - Configuration</name>
```

Comment:
I don't think we want a new module for that.

The tests should be added there: https://github.com/quarkusio/quarkus/tree/master/extensions/hibernate-orm/deployment/src/test/java/io/quarkus/hibernate/orm .